src/v2i-hub/MessageReceiverPlugin/manifest.json

```diff
@@ -29,9 +29,9 @@
 			"description":"Port for the incoming message network connection."
 		},
 		{
-			"key":"RouteDSRC",
+			"key":"RouteMessage",
```

Comment:
Why renaming this boolean?

Reply:
Is this boolean only for enable route for J2735Message? Some other internal v2xhub message like State, log on the UI are not controlled by this boolean? Maybe narrow down the Message to a specific type?

Reply:
Nope, just here. It's specific to some logic for broadcasting messages received by this plugin. 
I could change it to RouteJ2735

Reply:
make sense to me